cycler.py

```diff
@@ -563,3 +563,59 @@ def _cycler(label, itr):
         itr = (v[lab] for v in itr)
 
     return Cycler._from_iter(label, itr)
+
+
+def from_iter_of_dicts(inp):
+    """Construct a summation-only cycler from a list of dicts
+
+    Given an iterable of dictionaries (such as you would get from
+    iterating over a `Cycler`) and constructs a new `Cycler`.
+
+    The following are equivalent ::
+
+        from_iter_of_dicts(list(c)) == c.simplify()
+
+    Parameters
+    ----------
+    inp : Iterable[Mapping[Any, Any]]
```

Comment:
to be pedantic, wouldn't the first `Any` be `Hashable`? I haven't gotten fully into the annotation habit yet, so I don't know if `Mapping` already implies `Hashable` for the keys.